### ISSUE TITLE
Skin Placeholder position is not updating

### ISSUE THREAD

**Issue by michalochman** [pixijs/pixi-spine#230]
Hey, looks like the Skin Placeholder position is not updated when using the same region in different skins.

Take a look at the scenario I have prepared: https://codepen.io/michalochman/pen/XEWQgx

Try clicking red pixel:  Left app is working – the position is updated, Right app does not update the position.

1. First Atlas (broken) has 1 region called `pixel1`, second atlas (working) has 2 regions: `pixel1` and `pixel2` (they are duplicates with the same positions and dimensions).
2. Both JSON files have 2 skins, the only difference is that in the first JSON (broken) both skins are using the same region (`skin1 -> pixel1` / `skin2 -> pixel1`) and the second JSON (working) is using different regions (`skin1 -> pixel1` / `skin2 -> pixel2`).

**Comment by ivanpopelyshev** [comment]
try this thing:

```
mySpine.transformHack = function() { return 2; }
```

**Comment by ivanpopelyshev** [comment]
Nice, now both are wrong, lol.

**Comment by ivanpopelyshev** [comment]
Anyway, I'm very busy today and I doubt i can locate and fix this thing fast. Can you do some investigation, what causes this?

**Comment by michalochman** [comment]
@ivanpopelyshev looks like it's because of this condition:

[`if (!slot.currentSpriteName || slot.currentSpriteName !== ar.name) {`](https://github.com/pixijs/pixi-spine/blob/296c7039c107520e4bc0e0e4a3cc62545f970fca/src/Spine.ts#L225)

**Comment by michalochman** [comment]
Yes, they have the same name. In the example provided there's only one region. It is used in different skins that use placeholders with the same regions but different parameters (`y` in this case).
